web/src/components/notification-banner/notification-banner.tsx

```diff
@@ -42,28 +52,27 @@ export default function NotificationBanner({
   const [show, setShow] = useState(false);
 
   const el = useRef(null);
+  const { storageKey = '' } =
+    notification.kind == 'banner' && notification.bannerProps;
 
   useEffect(() => {
     setShow(true);
   }, []);
```

Comment:
I'm confused about what's happening here. It looks like we set the initial state to `false`, then immediately cause a rerender by updating it to `true`? Do you know why we're not just setting `const [show, setShow] = useState(true);` at the outset?

Reply:
Hmm, I think it's probably a hold-over from using banners as notifications.